package.json

```diff
@@ -13,7 +13,7 @@
     "watch-bundled": "web-ext run --watch-file dist/background.js"
   },
   "devDependencies": {
-    "@mozilla/rally": "^0.4.0",
+    "@mozilla/rally": "^0.5.0",
```

Comment:
Hey, did you use npm to update this dependency? For some reason the package-lock file was not updated automatically.

Reply:
Oops, sorry fixed.